src/adaptation/massmatrix.jl

```diff
@@ -171,13 +172,13 @@ function get_estimation(nc::NaiveCov)
 end
 
 # Ref: https://github.com/stan-dev/math/blob/develop/stan/math/prim/mat/fun/welford_covar_estimator.hpp
-mutable struct WelfordCov{F<:AbstractFloat} <: DenseMatrixEstimator{F}
+mutable struct WelfordCov{F<:AbstractFloat,C} <: DenseMatrixEstimator{F}
```

Comment:
For clarity, perhaps 

```suggestion
mutable struct WelfordCov{F<:AbstractFloat,C<:Matrix{F}} <: DenseMatrixEstimator{F}
```